scripts/rv_expression_association/prepare_inputs.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
+
+import click
+import logging
+import re
+import subprocess
+import sys
+import pandas as pd
+import xarray as xr
+from cloudpathlib import AnyPath
+from cpg_utils import to_path
+from cpg_utils.hail_batch import output_path
+
+subprocess.run([sys.executable, '-m', 'pip', 'install', 'limix==3.0.4'], check=True)
+subprocess.run(
+    [sys.executable, '-m', 'pip', 'install', 'pandas_plink==2.2.9'], check=True
+)
+
+from pandas_plink import read_plink1_bin  # lint complains about these but surely i need to install them before i can import them? 
```

Comment:
You're absolutely right. Maybe consider suppressing the linter warning here.

```suggestion
from pandas_plink import read_plink1_bin
```

Reply:
I didn't realise I could do that! How? I also have another recurrent lint comment about having "multiple modules named main" which I don't feel I can address really

Reply:
You can add comments to the offending lines, e.g.

```
# pylint: disable=locally-disabled
```

Where `locally-disabled` is the name of the warning you'd like to suppress.

Reply:
i am so sorry, I don't get why what I just changed doesn't work?

Reply:
I think it did work -- the remaining linter errors are about the *order* of other import statements, not the *position*:

```
scripts/rv_expression_association/prepare_inputs.py:4:0: C0411: standard import "import logging" should be placed before "import click" (wrong-import-order)
scripts/rv_expression_association/prepare_inputs.py:5:0: C0411: standard import "import re" should be placed before "import click" (wrong-import-order)
scripts/rv_expression_association/prepare_inputs.py:6:0: C0411: standard import "import subprocess" should be placed before "import click" (wrong-import-order)
scripts/rv_expression_association/prepare_inputs.py:7:0: C0411: standard import "import sys" should be placed before "import click" (wrong-import-order)
```

Reply:
mmmm am I looking at the wrong thing?

https://github.com/populationgenomics/tob-wgs/actions/runs/3141000124/jobs/5102974062

Reply:
The report in that link doesn't contain any warning for `wrong-import-position`, which is the category you've disabled. So I think this is working.

Reply:
sorry, I think my confusion comes from the fact that while pylint doesn't flag that problem anymore, flake8 still does? I didn't realise those two may sometime be overlapping, and couldn't understand why the problem was still there

Reply:
Oh, right -- there's unfortunately significant overlap between pylint and flake8. @illusional / @vladsavelyev Any thoughts on whether we should just pick one of them instead of running both?

Reply:
The latest template includes flake8, mypy, black, and pylint. Maybe reducing that to pylint and black would be sufficient?

Reply:
in the meantime, is there an easy way to disable flake8 for those same lines?

Reply:
sorry, does anyone have ideas re: flake8?

Reply:
and what about a way to fix the mypy error? 

```
> mypy.....................................................................Failed
> - hook id: mypy
> - exit code: 2
> 
> scripts/vep/main.py: error: Duplicate module named "main" (also at "scripts/generate_genotype_df/main.py")
> scripts/vep/main.py: note: Are you missing an __init__.py? Alternatively, consider using --exclude to avoid checking one of them.
> Found 1 error in 1 file (errors prevented further checking)
```

I think this is what Matt was trying to do, this has apparently been fixed elsewhere, but I am working off an older version? Sorry in advance if I am misunderstanding

Reply:
Did you see @MattWellie changes in https://github.com/populationgenomics/tob-wgs/pull/131/files to `.flake8`? Does that help?

Reply:
AFAIK this MyPy error will go away on main, as those two files have already been renamed there...

Reply:
ok perfect!

Reply:
> Did you see @MattWellie changes in https://github.com/populationgenomics/tob-wgs/pull/131/files to `.flake8`? Does that help?

does this need to be approved?

Reply:
> AFAIK this MyPy error will go away on main, as those two files have already been renamed there...

ok great!

Reply:
@annacuomo  I think #131 should be scrapped, best just to copy that one-line change into your branch

Reply:
> @annacuomo I think #131 should be scrapped, best just to copy that one-line change into your branch

aaaah I see! Sorry for being slow, should've done now

Reply:
ok i think i'm down to only the mypy error, which hopefully should be resolved already in main, is that right @lgruen ? Any more concerns?